src/Digdir.Domain.Dialogporten.Infrastructure/Altinn/Authorization/AltinnAuthorizationClient.cs

```diff
@@ -81,9 +81,8 @@ public async Task<AuthorizedPartiesResult> GetAuthorizedParties(IPartyIdentifier
         CancellationToken cancellationToken = default)
     {
         var authorizedPartiesRequest = new AuthorizedPartiesRequest(authenticatedParty);
-        // var authorizedParties = await _partiesCache.GetOrSetAsync(authorizedPartiesRequest.GenerateCacheKey(), async token
```

Comment:
Dette er ikke kommentert ut ved en feil. Dette - for en eller annen grunn ukjent for oss enda - er grunnen for at party lista _noen_ ganger ikke har med subparties, men _KUN på GQL_. Eller har jeg misforstått? Har vi funnet ut av hvorfor dette failer? 
cc @oskogstad

Reply:
I retrospekt burde vi lagt inn en kommentar på hvorfor vi kommenterte ut. 😅

Reply:
Aha, jeg bare gikk ut i fra at dette var en forglemmelse fra min side, burde ha brukt git blame. Da kan vel bare denne ligge inntil vi får funnet ut av det.